src/structures/GuildAuditLogs.js

```diff
@@ -575,6 +588,19 @@ class GuildAuditLogsEntry {
             { id: data.target_id },
           ),
         );
+    } else if (targetType === Targets.GUILD_SCHEDULED_EVENT) {
+      this.target =
+        guild.scheduledEvents.cache.get(data.target_id) ??
+        new GuildScheduledEvent(
+          guild.client,
+          this.changes.reduce(
+            (o, c) => {
+              o[c.key] = c.new ?? c.old;
+              return o;
+            },
+            { id: data.target_id },
```

Comment:
A removed guild scheduled event doesn't have the guild id in `this.changes`. This will result in the guild id being `undefined`.

```suggestion
            {
              id: data.target_id,
              guild_id: guild.id
            },
```